data/tokens.json

```diff
@@ -3520,35 +3524,13 @@
       }
     },
     "Top Bar": {
-      "gtmds_top-bar_shadow": {
-        "value": {
-          "x": "0",
-          "y": "1",
-          "blur": "2",
-          "spread": "0",
-          "color": "rgba({gtmds_color_blue-8}, 0.3)",
-          "type": "innerShadow"
-        },
-        "type": "boxShadow"
-      },
       "gtmds_menu_shadow": {
         "value": "{gtmds_shadow-md}",
         "type": "boxShadow"
       }
     },
     "Canvas Menu": {
-      "gtmds_canvas-menu_shadow-sm": {
-        "value": {
-          "x": "0",
-          "y": "1",
-          "blur": "1",
-          "spread": "0",
-          "color": "#5A7F9E22",
-          "type": "dropShadow"
-        },
-        "type": "boxShadow"
-      },
-      "gtmds_canvas-menu_shadow-md": {
+      "gtmds_canvas-menu_shadow": {
         "value": {
```

Comment:
I think this should be aligned with the shadow tokens and be an abstraction.

Reply:
It will be once shadow make it in, I am meeting with Sassy on that this week :)